src/FixerDefinition/FixerDefinition.php

```diff
@@ -36,9 +36,9 @@ public function __construct(
         $summary,
         $description,
         array $codeSamples,
-        $configurationDescription,
+        $configurationDescription = null,
         array $defaultConfiguration = null,
-        $riskyDescription
+        $riskyDescription = null
```

Comment:
I would really want to avoid optional parameters.
I used `array $defaultConfiguration = null` just because I cannot use `?array $defaultConfiguration`